mbq/atomiq/management/commands/atomic_run_consumer.py

```diff
@@ -84,9 +117,25 @@ def handle(self, *args, **options):
             consumer = Consumer(**consumer_kwargs)
 
             while self.signal_handler.should_continue():
-                consumer.run()
-                self.collect_metrics(**options)
-                self.run_delayed_cleanup(**options)
+                try:
+                    execution_start = arrow.utcnow().datetime
+                    processed_task = consumer.process_one_task()
+                    execution_end = arrow.utcnow().datetime
+                except Consumer.model.DoesNotExist:
+                    sleep(1)
+                except Exception:
+                    rollbar.report_exc_info()
```

Comment:
When we call `report_exc_info` does it also log the error? Rollbar is nice but logs are also nice

Reply:
I believe it does, i'll check